.github/workflows/pytest.yml

```
@@ -0,0 +1,36 @@
+name: Run Pytest
+
+on:
```

YAMLlint (1.38.0)

[warning] truthy value should be one of [false, true]

(truthy)